api/models/Media.js

```diff
@@ -144,3 +137,22 @@ const createAndAddSize = function (attrs) {
 
   return Media.create(attrs)
 }
+
+export function makeSubjectIdKeyAndValue ({subjectId, subjectType, postId, commentId}) {
+  let subjectIdKey, subjectIdValue
+
+  if (subjectId && subjectType) {
```

Comment:
im not quite clear why you need subjectId and subjectType alongside postId and commentId, cant you just use subjectId and subjectType for everything?